src/metaschema/oscal_ssp_metaschema.xml

```diff
@@ -705,8 +705,8 @@
       <field ref="remarks" in-xml="WITH_WRAPPER"/>
     </model>
     <constraint>
-      <allowed-values target="prop/@name" allow-other="yes">
-        <enum value="control-origination">Identifies the source of the implemented control.</enum>
+      <allowed-values target="(.|statement|.//by-component)/prop/@name" allow-other="yes">
+        <enum value="control-origination">Identifies the source of the implemented control.  Any <code>control-origination</code> prop defined in a child context will override the parent value.</enum>
       </allowed-values>
       <allowed-values target="prop[@name='control-origination']/@value">
```

Comment:
My concern in my previous comments was around the lack of specific values being defined. I missed the value set defined here because the path was not targeting the same nodes. This should fix it.
```suggestion
      <allowed-values target="(.|statement|.//by-component)/prop[@name='control-origination']/@value">
```